cc_corpus/corpus.py

```diff
@@ -363,3 +408,32 @@ def close(self):
     def __del__(self):
         """Just calls close()."""
         self.close()
+
+
+def write_document_to_json(document: Document, output_file: typing.TextIO):
```

Comment:
I would rather have a `convert_document_to_json()` function that does just the conversion and use it in-line when writing, i.e. `print(convert_document_to_json(document), file=...)`.

Reply:
Ok.